packages/components/rating/src/Rating.tsx

```diff
@@ -145,10 +145,10 @@ export const Rating = forwardRef<HTMLDivElement, RatingProps>(
           type="range"
           min="0"
           max="5"
-          step={1}
+          step={readOnly ? 0.5 : 1}
```

Comment:
```suggestion
          step={1}
```

Reply:
Sorry, but I don't understand your remark as it seems to be the opposite from what we discussed altogether yesterday :thinking:

Reply:
imho there is no point in updating the step, knowing that you cannot interact with the component in readOnly mode

Reply:
To me, the point was that we wanted to avoid rounding the value of the input (the value that is announced through voice over) when in non-interactive mode

Reply:
Just took the time to investigate on this, and it appears that the voice syntax (at least on Linux?) doesn't simply read the input value. He also seems to take into account the `step`, and in your case doesn't give the expected result. Happy to discuss, if some of you can also check with their voice syntax tool :+1:

Reply:
I'll take a look this afternoon or tomorrow 👍

Reply:
After testing it with voice over, you're right, we do need to update the **step** value. So, this PR looks good to me 👍